Assets/Resources/Materials/TransparentSlide.mat

```diff
@@ -7,7 +7,7 @@ Material:
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_Name: SimpleSlideColor1
+  m_Name: TransparentSlide
```

Comment:
TransparentSlideMaterial or Color? It's ok either way, just a thought so we do not confuse it with the slide gameObject